services/blue/src/api/DatabaseAccess/DbConnectionHolder.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading.Tasks;
+
+namespace artiso.AdsdHotel.Blue.Api
+{
+    internal class DbConnectionHolder : IDbConnectionHolder
+    {
+        private readonly IDbConnection _connection;
+        private IDbTransaction? _transaction;
+
+        public DbConnectionHolder(IDbConnection connection)
+        {
+            _connection = connection;
+        }
+
+        public bool HasTransaction => !(_transaction is null);
```

Comment:
God I hate that syntax. Can't wait for the not keyword ^^

Reply:
yep, but I've started to dislike the `!= null` in anticipation of the `not` operator... should we keep it as-is or use `!=`?

Reply:
the way to go atm would be `_transaction is object`

Reply:
and well, in 3 days you can use `not` :D

Reply:
I thought the only other option was `_transaction is { }` ... I'd say `is object` is acceptable